### ISSUE TITLE
Add support for other DTE speeds other than 9600

### ISSUE THREAD

**Issue by dranch** [fmarier/tmd710_tncsetup#2]
Your version of this tool and the copy I maintain both are hard coded to use a DTE speed of 9600.  It might be nice to add a command line option to support other speeds.  The D710 supports 9600, 19200, 38400, and 57600.  Realistically, supporting anything more than 19200 doesn't make much sense since as even FSK9600 doesn't offer any form of compression thus no speed up but it would be nice to offer this for compatibility.

**Comment by fmarier** [comment]
Good idea. I don't have a D710, so I would need someone with that radio to propose and test this change.

**Comment by dranch** [comment]
Ha.. ok.  I have a D710 and can test for you

**Comment by dranch** [comment]
Any updates here to see if an additional command line option could be offered for faster DTE rates?

**Comment by fmarier** [comment]
Since I have no way to test this myself, I'll defer to someone with such hardware to propose a pull request.

**Comment by dranch** [comment]
If you can send me a patch, I can test

**Comment by fmarier** [comment]
I'm not sure exactly what you'd like this command-line option to be. Is it:

- extending the existing `--baud-rate` option to support more than just 1200 or 9600 baud, or
- adding a new option to make the `SERIAL_SPEED` configurable?

**Comment by dranch** [comment]
Make the SERIAL_SPEED configurable.  Currently it's hard coded to 9600bps between the computer and the TNC.  I'm not talking about the RF speed here
